### ISSUE TITLE
Latest version does not detect mouse on windows

### ISSUE THREAD

**Issue by agusmba** [xournalpp/xournalpp#1639]
**Affects versions :**
 - OS: Windows 10
 - Version of Xournal++ [1.1.0+dev](https://dev.azure.com/xournalpp/xournalpp/_build/results?buildId=3576&view=artifacts)

**Describe the bug**
After a new install, nothing can be edited. It turns out that the mouse is not recognized correctly as an input device:

**To Reproduce**
1. Install the latest dev version
2. open Xournal++
3. Try to edit anything, nothing changes on the blank page

**Expected behavior**
After opening the program, it should be able to edit a new document.

**Screenshots of Problem**

![image](https://user-images.githubusercontent.com/3104020/69888213-fa58eb00-12ea-11ea-8841-9dd640fca9de.png)

**Additional context**

This can be *fixed* by going into preferences and manually selecting the mouse as input device, as discussed in #1600

**Comment by LittleHuba** [comment]
I assume this "mouse" is matched as a virtual pointer which is ignored as it generates problems on Linux. Hopefully disabling this check for Windows will solve this issue. I'll provide a merge in a few hours once I finished another one. Please set up your system to build Xournal++ manually (see our wiki). You'll have to test if it works.

**Comment by LittleHuba** [comment]
Okay I need an input debug log from you to fix this. Please follow the instructions in the wiki to generate one.

**Comment by agusmba** [comment]
I was thinking about using github actions with azure-pipelines to build the debug builds on a fork from this repo, but I just saw that you already have some azure-pipelines configuration done. Are those builds available for testing purposes?

**Comment by LittleHuba** [comment]
Yeah you can simple make a PR on our repository and the pipelines will kick in. You should be able to see the results of the runs as they are public.

**Comment by agusmba** [comment]
Ok, thanks! will those runs include the installer as an artifact?
I was looking around and it seemed that the debug builds were used for testing but I was not seeing the downloadable artifacts (only for release pipelines, but those aren't debug-enabled, right?).

**Comment by LittleHuba** [comment]
No the installer is only created for releases. The CI pipelines are not meant for this. It is better if you generate the installer locally on your system.

**Comment by agusmba** [comment]
> No the installer is only created for releases. The CI pipelines are not meant for this. It is better if you generate the installer locally on your system.

Understood, I'll not pollute the main repo with debug PRs, I'll try to set up a build box locally.

**Comment by LittleHuba** [comment]
Setting it up is not very hard. Just follow the wiki entry.

**Comment by agusmba** [comment]
Ok, after the debug run, I get:

```
** (xournalpp.exe:5240): WARNING **: 15:54:12.416: Failed to load "colornames.ini" (C:\msys64\home\IEUser\.xournalpp\\colornames.ini): No such
file or directory


** (xournalpp.exe:5240): WARNING **: 15:54:12.889: configfile does not exist C:\msys64\home\IEUser\.xournalpp\settings.xml


** (xournalpp.exe:5240): WARNING **: 15:54:14.242: No device found. Is Xournal++ running in debugger / Eclipse...?
Probably this is the reason for not finding devices!


(xournalpp.exe:5240): Pango-WARNING **: 15:54:18.581: couldn't load font "C059 Not-Rotated 8", falling back to "Sans Not-Rotated 8", expect ugl
y output.
** Message: 15:54:19.480: Event
Event type:     GDK_FOCUS_CHANGE
Source device:  GDK_SOURCE_KEYBOARD
Device Class:   INPUT_DEVICE_KEYBOARD

** Message: 15:54:21.023: Event
Event type:     GDK_ENTER_NOTIFY
Source device:  GDK_SOURCE_MOUSE
Device Class:   INPUT_DEVICE_MOUSE

** Message: 15:54:21.039: Event
Event type:     GDK_MOTION_NOTIFY
Source device:  GDK_SOURCE_MOUSE
Device Class:   INPUT_DEVICE_IGNORE

** Message: 15:54:22.151: Event
Event type:     GDK_BUTTON_PRESS
Source device:  GDK_SOURCE_MOUSE
Device Class:   INPUT_DEVICE_IGNORE
Button: 1

** Message: 15:54:22.216: Event
Event type:     GDK_MOTION_NOTIFY
Source device:  GDK_SOURCE_MOUSE
Device Class:   INPUT_DEVICE_IGNORE

...

```

**Comment by LittleHuba** [comment]
Okay I'll have to write a special version that outputs some more values so I can determine the specific cause. I'll write you the branch name here once I'm done. Please upload again the console output when running this version.
